host/src/components/GameInProgressContentSwitch.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
```

Comment:
Because the teacher is able to add and remove features on a per-question basis, we need a way to handle what gets render at the gameplay level. This container component determines what gets rendered on the screen based on how the teacher has configured each question and if a teacher has clicked on a graph.